<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmployeeDialog</class>
 <widget class="QDialog" name="AddEmployeeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblType">
     <property name="text">
      <string>Employee &amp;type:</string>
     </property>
     <property name="buddy">
      <cstring>cbType</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbType">
     <item>
      <property name="text">
       <string>Employee</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manager</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sales</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Employee &amp;name:</string>
     </property>
     <property name="buddy">
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>&amp;Date of employeed:</string>
     </property>
     <property name="buddy">
      <cstring>deDateOfEmployed</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="deDateOfEmployed">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2016</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblSalary">
     <property name="text">
      <string>Base &amp;salary:</string>
     </property>
     <property name="buddy">
      <cstring>sbBaseSalary</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbBaseSalary">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEmployeeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEmployeeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
